courses/free-courses-ta.md

```diff
@@ -50,6 +51,8 @@
 CSE Tamila by Eezytutorials
 * [Data Structures and Algorithms in Java](https://www.youtube.com/playlist?list=PLYM2_EX_xVvX7_AmNY-Deacp3rT3MIXnE) - Logic First Tamil
 
+### <a id="ethical-hacking"></a>Ethical Hacking
+* [Learn Ethical Hacking in Tamil](https://www.youtube.com/watch?v=Gn6hYOAMB5k&list=PL2fEqBvRZvbfWhFw_YiaY_0Fv8e8ouNkP) - Ethical Hacking Playlist by Tamil Hacks 2.0
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. In other lists, 'ethical hacking' resources are listed under "security"